screens/ShopSearch.js

```diff
@@ -63,9 +67,9 @@ export default class ShopSearch extends React.Component {
                   <Text style={styles.searchText}>How big is your mail load?</Text>
                 </View>
                 <View style={{flexDirection: 'row'}}>
-                  <Text style={styles.quantityText}>{this.state.pickerSelection}</Text>
+                  <Text style={styles.quantityText}>{this.state.pickerSelection1}</Text>
                   <View style={{position: 'absolute', marginTop: window.height*.05, marginLeft: window.width*.7}}>
-                    <DropDown selectedValue={this.state.pickerSelection} setState={this.setPicker}/>
+                    <DropDown selectedValue={this.state.pickerSelection1} setState={this.setPicker1} label1={'Heavy'} label2={'Normal'} label3={'Light'} value1={'Heavy'} value2={'Normal'} value3={'Light'}/>
```

Comment:
Again, you should only have one prop that takes in an array of labels and values instead of having six different props.